src/main/java/com/puppycrawl/tools/checkstyle/checks/TodoCommentCheck.java

```diff
@@ -50,13 +50,26 @@
  * &lt;module name="TodoComment"/&gt;
  * </pre>
  * <p>
+ * Example:
+ * </p>
+ * <pre>
+ * //TODO: ..... violation
+ * </pre>
+ * <p>
  * To configure the check for comments that contain {@code TODO} and {@code FIXME}:
  * </p>
  * <pre>
  * &lt;module name="TodoComment"&gt;
  *   &lt;property name="format" value="(TODO)|(FIXME)"/&gt;
  * &lt;/module&gt;
  * </pre>
+ * <p>
+ * Example:
+ * </p>
+ * <pre>
+ * //TODO: ..... violation
+ * //FIXME: ..... violation
```

Comment:
Please add cases without violation

Reply:
Ok  i will add two now

Reply:
please do not reply "I will", please reply "done" ... to let me catch notification that your are ready for review.

Reply:
you are right
done